src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/action/FetchAction.java

```diff
@@ -36,15 +38,21 @@
     private GerritUtil gerritUtil;
     @Inject
     private GerritGitUtil gerritGitUtil;
-
+    @Inject
+    private NotificationService notificationService;
 
     public void fetchChange(ChangeInfo selectedChange, final Project project, final Callable<Void> successCallable) {
         gerritUtil.getChangeDetails(selectedChange._number, project, new Consumer<ChangeInfo>() {
             @Override
             public void consume(ChangeInfo changeDetails) {
 
                 Optional<GitRepository> gitRepository = gerritGitUtil.getRepositoryForGerritProject(project, changeDetails.project);
-                if (!gitRepository.isPresent()) return;
+                if (!gitRepository.isPresent()) {
+                    NotificationBuilder notification = new NotificationBuilder(project, "Error",
```

Comment:
I'm not sure neither, but I tend to say that these few duplicate lines are below the threshold that would justify a separate method.